src/System.Text.Json/src/System/Text/Json/Serialization/JsonIgnoreAttribute.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Text.Json.Serialization
+{
+    /// <summary>
+    /// Prevents a property from being serialized or deserialized.
```

Comment:
The type that uses the attribute is called `JsonSerializer` tho 😋

Reply:
Sure, but the term "deserialized" doesn't appear anywhere:
https://github.com/dotnet/corefx/blob/ce62a086814fdfecea1d6ba6657c3fb88763e1d6/src/System.Text.Json/ref/System.Text.Json.cs#L313-L327

The `JsonSerializer` is the type (noun), what it does is read/write/parse (verb) :)

My feedback is that we should be clearer in our docs so users understand what verb/operation is happening and how this attribute changes that (or at least be consistent).

Reply:
A serializer serializes. It may read from a stream or parse a string when it is serializing.

A serializer deserializes. It may write to a stream, or convert the output to bytes or a string when it is deserializing.

Reply:
OK, that's fair.